.github/workflows/codeql-analysis.yml

```
@@ -66,22 +54,37 @@
+        #define PJMEDIA_AUDIO_DEV_HAS_WASAPI 0
+        #define PJMEDIA_AUDIO_DEV_HAS_WMME 0
+        EOL
+
+        cat ./pjlib/include/pj/config_site.h
+
+        ./configure
+        make realclean
+        make dep
+        make
+        sudo make install
+        popd 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)